components/logged_in_navigation/components/notifications/view.coffee

```diff
@@ -6,15 +6,22 @@ module.exports = class NotificationsView extends Backbone.View
   events:
     'click a': -> delay (=> @render()), 250
 
-  initialize: ->
+  initialize: ({ @count }) ->
     @listenTo @collection, 'sync', @render
+    @listenTo @count, 'change:unread_count', @render
 
     @$el
-      .one 'mouseenter', => @collection.markRead()
+      .one 'mouseenter', =>
+        @collection.fetch()
```

Comment:
What does this actually look like? Is it just blank?

Reply:
![hinotif](https://user-images.githubusercontent.com/112297/38949638-f98eb2d2-4310-11e8-84ac-123aee1aa7f1.gif)

Reply:
Wow, surprising 👍